brainframe/cli/translations/update.en.yml

```diff
@@ -13,14 +13,15 @@ en:
   complete: "BrainFrame has been updated!"
   upgrade-version: "BrainFrame:%{existing_version} detected under your installation
   location, we will replace it with BrainFrame:%{requested_version}"
+  version-failing: "You have BrainFrame:%{existing_version} installed already.
+  The latest version we found on updating server is
+  BrainFrame:%{upgrade_version}. We don't recommend downgrading BrainFrame
+  for data integrity reasons. This request will be ignored. But you can force to
+  downgrade by appending --downgrade flag."
```

Comment:
```suggestion
  for data integrity reasons. This request will be ignored. But you can force a
  downgrade by also supplying the --downgrade flag."
```

Reply:
Suffice to say I did not write most of these messages 😅 Thanks for the fixes!